# OpenStax Chemistry, 2nd Ed. &ndash; Dean's Errata Submissions

## Chapter 4

### Chapter 4 &ndash; Exercises

In [2]:
# Execute this cell to access OpenStax Chem 2e book
# atomic weights with b.* and IUPAC weights with .*
from p import *

**Execute this cell to display more chemical formulas.**
$$ \require{mhchem} $$

#### Ex 4.61

$$\text{2Na} (s) + \text{Cl}_2 (g) \rightarrow \text{2NaCl}$$

In [2]:
8 / b.Na

0.34797738147020446

In [3]:
8 / (b.Cl*2)

0.11283497884344146

$$\frac{0.35 \text{ mol Na}}{0.11 \text{ mol Cl}_2} > \frac21 \text{ so Cl}_2 \text{ is limiting reactant.}$$

#### Ex 4.63

In [4]:
25 / 81 * 100

30.864197530864196

$$31 \% \text{ yield}$$

#### Ex 4.65

1. Balance the chemical formula.
2. Calculate moles of CCl<sub>4</sub> from mass and formula weight.
3. Calculate moles CCl<sub>2</sub>F<sub>2</sub> using stoichoimetric factor.
4. Calculate mass of CCl<sub>2</sub>F<sub>2</sub>, which is the theoretical yield.
5. If not done before, combine the calculations into a single calculation to avoid intermediate rounding.
6. Divide actual yield given by theoretical yield from previous step (and multiply by 100) to get % yield.
$$\text{CCl}_4 + 2\text{HF} \rightarrow \text{CCl}_2\text{F}_2 + 2\text{HCl}$$

In [5]:
12.5 / (32.9 / (b.C + b.Cl*4) * 1 / 1 * (b.C + b.Cl*2 + b.F*2)) * 100

48.33218910596668

$$48.3\% \text{ yield}$$

#### Ex 4.67

$$2\text{C}_6\text{H}_5\text{CH}_3 + 3\text{O}_2 \rightarrow 2\text{C}_6\text{H}_5\text{CO}_2\text{H} + 2\text{H}_2\text{O} $$

What is the percent yield of a reaction that converts 1.000 kg of toluene to 1.21 kg of benzoic acid?

In [6]:
1.21 * 10**3 / (1.000 * 10**3 / (b.C*6 + b.H*5 + b.C + b.H*3) * 2 / 2 * (b.C*6 + b.H*5 + b.C + b.O*2 + b.H)) * 100

91.29050590412552

$$91.3\% \text{ yield}$$

#### Ex 4.69

1. Start with L of ethanol.
2. Times density of ethanol to calculate g of ethanol.
3. Divide by formula mass of ethanol to get moles of ethanol.
4. Apply stoichiometric factor to get moles of ether.
5. Times formula mass of ether to get g of ether.
6. Divide by density of ether to get L of ether theoretical.
7. Divide L of ether actual by L of ether theoretical to get % yield.

In [7]:
1.17 / (1.500 * 10**3 * 0.7894 / (b.C*2 + b.H*5 + b.O + b.H) * 1 / 2 * ((b.C*2 + b.H*5)*2 + b.O) / 0.7134 / 10**3) * 100

87.62429339312541

$$87.6\% \text{ yield}$$

#### Ex 4.71

1. Calculate the provided ratio of reactants by comparing 0.50 mol Cr to 0.75 mol H<sub>3</sub>PO<sub>4</sub>.
2. Calculate the required ratio of reactants by comparing the number of Cr molecules 2 in the balanced formula to the number of H<sub>3</sub>PO<sub>4</sub> molecules, also 2.
3. If the provided ratio is higher, H<sub>3</sub>PO<sub>4</sub> is the limiting reactant, otherwise Cr is the limiting reactant.


In [8]:
0.50 / 0.75

0.6666666666666666

In [9]:
2 / 2

1.0

$$ .67 < 1.0 \text{ , so Cr is the limiting reactant.}$$

#### Ex 4.73

0.4031 g sodium oxalate Na<sub>2</sub>C<sub>2</sub>O<sub>4</sub> is ? moles

1.481 g uranyl nitrate UO<sub>2</sub>(NO<sub>3</sub>)<sub>2</sub> is ? moles

In [10]:
0.4031 / (b.Na*2 + b.C*2 + b.O*4)

0.0030082089552238805

In [11]:
1.481 / (b.U + b.O*2 + (b.N + b.O*3)*2)

0.0037586924521597893

So calculating the theoretical yield limited by sodium oxalate, and using actual 1.073 g output of (long formula) with a 1 to 1 stoichiometric factor with sodum oxalate, will let us calculate the percent yield.

In [12]:
1.073 / (0.4031 / (b.Na*2 + b.C*2 + b.O*4) * 1 / 1 * (b.U + b.O*2 + b.C*2 + b.O*4 + 3*(b.H*2 + b.O))) * 100

86.56111308861995

$$\text{sodium oxalate is limiting. } 86.56\% \text{ yield}$$

#### Ex 4.75

Saccharin is C<sub>7</sub>H<sub>5</sub>NO<sub>3</sub>S.

Taking that ratio and comparing it to the number of atoms available in the problem gives:

7:5:1:3:1, compared to
30:25:14:12:8

In [13]:
print(f'C could make {30 / 7} saccharin molecules')
print(f'H could make {25 / 5} saccharin molecules')
print(f'N could make {14 / 1} saccharin molecules')
print(f'O could make {12 / 3} saccharin molecules')
print(f'S could make {8 / 1} saccharin molecules')

C could make 4.285714285714286 saccharin molecules
H could make 5.0 saccharin molecules
N could make 14.0 saccharin molecules
O could make 4.0 saccharin molecules
S could make 8.0 saccharin molecules


So O is the limiting component, and there is enough to make 4 saccharin molecules.

#### Ex 4.77

The internet says gold is currently $5,053.11 / troy ounce.  The problem states

> Would you agree to buy 1 trillion (1,000,000,000,000) gold atoms for $5? Explain why or why not. ... (1 troy ounce=31.1 g)

In [14]:
AvogadrosNumber = 6.022 * 10**23
10**12 # atoms
10**12 / AvogadrosNumber # moles
10**12 / AvogadrosNumber * b.Au # grams
10**12 / AvogadrosNumber * b.Au / 31.1 # troy ounces
10**12 / AvogadrosNumber * b.Au / 31.1 * 5053.11 # dollars


5.315251740403088e-08

Answer: Yes, I would by 1 trillion gold atoms for 5 dollars, if I could use a microscope good enough to verify they were present.  However, that would just be for a conversation piece so I could say I owned 1 trillion gold atoms.  It would not be close to worth it as an investment, being worth a tiny fraction of a penny.

#### Ex 4.79

$$\text{H}_2\text{SO}_4 \text{ (aq)} + 2\text{NaOH} \rightarrow \text{Na}_2\text{SO}_4 \text{ (aq)} + 2\text{H}_2\text{O} \text{ (l)}$$

In [15]:
1.7  # mL NaOH solution
1.7 / 10**3  # L NaOH solution
1.7 / 10**3 * 0.0811  # mol NaOH
1.7 / 10**3 * 0.0811 * 1 / 2  # mol H2SO4
1.7 / 10**3 * 0.0811 * 1 / 2 / (20 / 10**3) # mol/L H2SO4

0.00344675

3.4 x 10<sup>-3</sup> M H<sub>2</sub>SO<sub>4</sub>

#### Ex 4.81

What is the Cl− concentration in a 0.25-mL sample of normal serum that requires 1.46 mL of 8.25 × 10−4 M Hg(NO3)2(aq) to reach the end point?

In [16]:
1.46  # mL Hg(NO3)2 solution
8.25 * 10**-4  # M Hg(NO3)2 = mol / L or mmol / mL
1.46 * 8.25 * 10**-4  # mmol Hg(NO3)2
1.46 * 8.25 * 10**-4 * 2 / 1 # mmol Cl+
1.46 * 8.25 * 10**-4 * 2 / 1 / 0.25 # mmol / mL = M Cl+

0.009636

9.6 x 10<sup>-3</sup> M Cl<sup>-</sup>

#### Ex 4.83

> A sample of gallium bromide, GaBr<sub>3</sub>, weighing 0.165 g was dissolved in water and treated with silver nitrate, AgNO<sub>3</sub>, resulting in the precipitation of 0.299 g AgBr. Use these data to compute the %Ga (by mass) GaBr<sub>3</sub>.

$$
\text{GaBr}_3 \text{ (aq)} +
\text{AgNO}_3 \text{ (aq)} +
\rightarrow
\text{AgBr (s)} +
\text{GaNO}_3
$$

In [17]:
b.Ga / (b.Ga + b.Br*3) * 100

22.53248012410316

In [18]:
i.Ga / (i.Ga + i.Br*3) * 100

22.532357360996656

In [19]:
0.299  # g AgBr
0.299 / (b.Ag + b.Br)  # mol AgBr
0.299 / (b.Ag + b.Br) * 1 / 1  # mol GaBr3
0.299 / (b.Ag + b.Br) * 1 / 1 * (b.Ga + b.Br*3)  # g GaBr3

0.4926335463258786

$\Large?$

Honestly, I just don't get this one.  I can calculate the % of Ga by mass in GaBr<sub>3</sub> without any of that data, just the formula GaBr<sub>3</sub> and the atomic weights of Ga and Br.  I assume I am not supposed to do that.  However, if I don't use any atomic weights of anything, it seems like it would be degenerate, and I cannot solve it at all with just this data.  For example, let Mg be mass of Ga, Mb be mass of Br, and Ma be mass of Ag (silver):

$$ Mg + Mb = 0.165 g $$
$$ Ma + Mb = 0.299 g $$

A system of two equations with three unknowns.

$$ Mb = 0.299 - Ma $$
$$ Mg = 0.165 - Mb = 0.165 - (0.299 - Ma) $$

But I don't know Ma, the mass of silver.  Either I'm missing something or their wording is unclear or ?

#### Ex 4.85

A 0.025-g sample of a compound composed of boron and hydrogen, with a molecular mass of ~28 amu, burns spontaneously when exposed to air, producing 0.063 g of B2O3. What are the empirical and molecular formulas of the compound?

$$
a \space \text{B}_x \text{H}_y \space (0.025 g) +
b \space \text{O}_2
\rightarrow 
c \space \text{B}_2 \text{O}_3 \space (0.063 g) +
d \space \text{H}_2 \text{O} \space ?
$$

In [20]:
print(f'Boron atomic weight: {b.B}')
print(f'Hydrogen atomic weight: {b.H}')
print(f'Oxygen atomic weight: {b.O}')
print(f'0.025 / 0.063: {0.025 / 0.063}')
print(f'Formula mass of B2O3: {b.B*2 + b.O*3} amu')

Boron atomic weight: 10.81
Hydrogen atomic weight: 1.008
Oxygen atomic weight: 16.0
0.025 / 0.063: 0.39682539682539686
Formula mass of B2O3: 69.62 amu


$$ 10.81x + 1.008y \approx 28 \text{ amu (limiting x to 1 or 2)}$$
$$ d = 2b $$
$$ ax = 2c $$
$$ ay = 2d $$
$$ \frac{a\space(10.81x + 1.008y)}{c\space(10.81\cdot2 + 16.00\cdot3)} = \frac{0.025g}{0.063g} $$
$$ \frac{a\space(10.81x + 1.008y)}{c\space(69.62)} = 0.397 $$
Wait
$$ \frac{a\space(\sim 28 \text{ amu})}{c\space(10.81\cdot2 + 16.00\cdot3 \text{ amu})} = \frac{0.025g}{0.063g} $$
$$ \frac{a\space(\sim 28 \text{ amu})}{c\space(69.62 \text{ amu})} = 0.397 $$


In [21]:
28 / 69.62


0.40218328066647513

$$ \therefore a = c $$
So
$$ a = c $$
$$ d = 2b $$
$$ ay = 2d $$
$$ ax = 2c $$
$$ \therefore x = 2 $$

$$
a \space \text{B}_2 \text{H}_y \space (0.025 g) +
b \space \text{O}_2
\rightarrow 
c \space \text{B}_2 \text{O}_3 \space (0.063 g) +
d \space \text{H}_2 \text{O} \space ?
$$

$$ \frac{a\space(10.81\cdot2 + 1.008y)}{a\space(10.81\cdot2 + 16.00\cdot3)} = \frac{0.025g}{0.063g} $$
$$ \frac{\cancel{a}\space(10.81\cdot2 + 1.008y)}{\cancel{a}\space(69.62)} = 0.397 $$
$$ 10.81\cdot2 + 1.008y = 0.397 \cdot 69.62$$

In [22]:
10.81 * 2

21.62

In [23]:
0.397 * 69.62

27.639140000000005

$$ 21.62 + 1.008y = 27.639 $$
$$ 1.008y = 27.639 - 21.62 $$
$$ y = \frac{27.639 - 21.62}{1.008} $$

In [24]:
(27.639 - 21.62) / 1.008

5.971230158730157

$$ x = 2 $$
$$ y = 6 $$
$$ \text{molecular formula is B}_2 \text{H}_6 $$
$$ \text{empirical formula is BH}_3 $$

#### Ex 4.87

In [25]:
2.50  # g NaHCO3
2.50 / (b.Na + b.H + b.C + b.O*3)  # mol NaHCO3
2.50 / (b.Na + b.H + b.C + b.O*3) * 1 / 1 # mol HCl
2.50 / (b.Na + b.H + b.C + b.O*3) * 1 / 1 / 0.600 # L HCl solution

0.04959845094118021

0.0496 L HCl, or 49.6 mL HCl

#### Ex 4.89

In [26]:
15.00  # mL C2O4H2 solution
15.00 * 0.1500  # mmol C2O4H2
15.00 * 0.1500 * 2 / 1  # mmol NaOH
15.00 * 0.1500 * 2 / 1 / 0.3300 # ml NaOH solution



13.636363636363635

13.64 mL NaOH solution

#### Ex 4.91

In [27]:
36.6  # ml HCl solution
36.6 * (5.00 * 10**-2)  # mmol HCl
36.6 * (5.00 * 10**-2) * 1 / 2  # mmol Ca(OH)2
36.6 * (5.00 * 10**-2) * 1 / 2 / 75.00 # mmol/mL = M Ca(OH)2 solution

0.0122

1.22 x 10<sup>-2</sup> M

#### Ex 4.93

In [28]:
40.00  # ml H3PO4 solution
40.00 * 0.0656  # mmol H3PO4
40.00 * 0.0656 * 2 / 1 # mmol KOH
40.00 * 0.0656 * 2 / 1 / 0.1500 # ml KOH solution

34.98666666666667

34.99 mL KOH solution

#### Ex 4.95

The reaction of WCl6 with Al at ~400 °C gives black crystals of a compound containing only tungsten and chlorine. A sample of this compound, when reduced with hydrogen, gives 0.2232 g of tungsten metal and hydrogen chloride, which is absorbed in water. Titration of the hydrochloric acid thus produced requires 46.2 mL of 0.1051 M NaOH to reach the end point. What is the empirical formula of the black tungsten chloride?

$$ ? \text{ WCl}_6 + ? \text{ Al} \rightarrow ? \text{ W}_x\text{Cl}_y + ? $$
$$ \text{W}\text{Cl}_4 + 2 \text{H}_2 \rightarrow \text{ W} + 4 \text{HCl (aq)} $$
$$ \text{HCl (aq)} + \text{NaOH (aq)} \rightarrow \text{NaCl (aq)} + \text{H}_2\text{O (l)} $$

In [29]:
46.2  # mL NaOH solution
46.2 * 0.1051  # mmol NaOH
46.2 * 0.1051 * 1 / 1 # mmol HCl

4.85562

In [30]:
0.2232  # g W
0.2232 / b.W  # mol W
0.2232 / b.W * 10**3 # mmol W

1.2143634385201305

In [31]:
(46.2 * 0.1051 * 1 / 1) / (0.2232 / b.W * 10**3)  # mmol HCl / mmol W

3.9984899462365595

$$ \text{WCl}_4 \text{ is the empirical formula of black tungsten chloride} $$

In [3]:
b.N

14.01

In [4]:
i.N

14.007